currency-rates/internal/handlers/subscribe.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"regexp"
 
-	"go.uber.org/zap"
+	"github.com/RinaDish/currency-rates/tools"
 )
 
 var emailRegex = regexp.MustCompile(`^[a-zA-Z0-9._%+\-]+@[a-zA-Z0-9.\-]+\.[a-zA-Z]{2,}$`)
```

Comment:
MustCompile can return panic. Also this is global variable. Maybe better to make from it separate package like, email validator witch you will initialize in main, so it will be more controlled?

Reply:
As I understand it, this would cause a panic during compilation and the variable is not global. Therefore, I suggest leaving it as it is)